apps/portal-api/src/migrations/20251216090105_update_service_instance_tags_openaev.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+import { v4 as uuidv4 } from 'uuid';
+
+export async function up(knex) {
+  const serviceInstanceId = uuidv4();
+  const linkServiceDefinition = await knex('ServiceDefinition')
+    .where('identifier', '=', 'link')
+    .select('id')
+    .first();
+  if (!linkServiceDefinition?.id) {
+    throw new Error('Service definition not found');
+  }
+
+  await knex('ServiceInstance').insert({
+    id: serviceInstanceId,
+    name: 'OpenAEV 101',
+    description:
+      'The fundamentals: discover the essential concepts with basic use cases and workflows.',
```

Comment:
This description may be the same as the academy introduction:

```
Discover how to deploy and use OpenAEV, Filigran's platform for realistic cyber-threat simulation, to build hands-on adversary emulation exercises.
```

That's what we done with OpenCTI. It should be confirmed with @EllynBsc